test/Microsoft.Azure.WebJobs.Host.EndToEndTests/WebJobs.Host.EndToEndTests.csproj

```diff
@@ -36,6 +36,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <LangVersion>default</LangVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>..\..\src\Common\PublicKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
Well, I'm at a loss for why this was able to build before. But we don't have AppVeyor building PRs yet so my guess is it would have failed on my previous PR without it -- I must have set it up that way locally and forgot to commit before I left on vacation a couple weeks ago. 

Long story short -- we need `InternalsVisibleTo` so we can simulate two different machines, so it's required.